content/en/cloudcraft/getting-started/datadog-integration.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Datadog Integration
+---
+
+## Overview
+
+The integration between Datadog and Cloudcraft provides users with a streamlined workflow for monitoring and visualizing their cloud infrastructure.
+
+By leveraging Datadog’s powerful monitoring platform, users can log in to Cloudcraft with their Datadog account, move seamlessly from any resource in Cloudcraft to the most relevant views in Datadog, and automatically synchronize cloud accounts that have already been configured in Datadog for use in Cloudcraft.
+
+## Datadog Single Sign-On (SSO)
+
+Cloudcraft allows users to sign up and log in using their Datadog account. This integration offers a unified experience, linking your Datadog monitoring data seamlessly with your Cloudcraft architecture diagrams.
```

Comment:
```suggestion
Cloudcraft allows users to sign up and log in using their Datadog account. This integration offers a unified experience, linking your Datadog monitoring data with your Cloudcraft architecture diagrams.
```
"Seamlessly" appears in the section immediately above, so just looking to avoid overuse.